pkg/server/ca/manager.go

```diff
@@ -315,7 +320,32 @@ func (m *Manager) prepareJWTKey(ctx context.Context, slot *jwtKeySlot) (err erro
 		return err
 	}
 
-	if err := m.appendBundle(ctx, nil, publicKey); err != nil {
+	publicKeys := []*common.PublicKey{publicKey}
+	upstreamAuth, useUpstream := m.c.Catalog.GetUpstreamAuthority()
+	if useUpstream {
+		publishCtx, cancel := context.WithTimeout(ctx, 5*time.Second)
```

Comment:
Oh yeah, I will do it. Since there is another place where we use this same value, do you think it worth to have only one constant for both usages? If so, where do you think that constant should be placed?

Reply:
Never mind, I have an idea

Reply:
Just added the constant in server.go, and passed it as needed. This way only one constant is defined.